learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,94 @@
+# Integrate Meilisearch Cloud with Vercel
+
+Getting a Meilisearch Cloud instance created and linked to your Vercel projects has never been easier. 
+
+## Introducing our tools
+
+### What is Vercel?
+
+[Vercel](https://vercel.com/) is a cloud platform for building and deploying web applications. It works out of the box with most popular web development tools.
+
+### What is Meilisearch Cloud?
+
+[Meilisearch Cloud](https://www.meilisearch.com/cloud) is a cloud-based SaaS platform, built on top of the Meilisearch open-source search engine.
```

Comment:
> cloud based SaaS platform, built on top of meilisearch oss search engine

This is very descriptive. Can we make it more "marketingy", like a few keywords that recaps the main stengthes of the cloud